### PR TITLE
show first and first Letter of last name

### PR DESCRIPTION
**Pull request by HennyHaelbich**
solves #62

### CODE REVIEW

**Review by benjamin-ranft — APPROVED** [review]
LGTM, did you add integration tests for your controller yet?

File: backend/src/main/java/de/neuefische/henny/reisekasse/service/UserService.java [hunk 1]
```diff
@@ -18,8 +21,13 @@ public UserService(UserDb userDb) {
...
     }
 
-    public Optional<TravelFoundUser> getUserById(String username) {
-        return userDb.findById(username);
+    public Optional<UserDto> getUserById(String username) {
+        Optional<TravelFoundUser> optionalUser = userDb.findById(username);
+        if(optionalUser.isEmpty()) {
+            return Optional.empty();
+        }
+        UserDto userWithoutPassword = new UserDto(optionalUser.get().getUsername(), optionalUser.get().getFirstName(), optionalUser.get().getLastName());
```
Comment (fabianschmauder) [inline backend/src/main/java/de/neuefische/henny/reisekasse/service/UserService.java:29]: you could create a variable `TravelFoundUser  user = optionalUser.get() `

File: frontend/src/addEventPage/useEventMembers.js [hunk 2]
```diff
@@ -14,7 +15,8 @@ const { token } = useContext(LoginContext);
   const addMember = (member) =>
     axios
       .get('/api/users/' + member, header(token))
-      .then((response) => response.data);
-  
-  return [addMember]
+      .then((response) => response.data)
+    //  .then((data) => console.log("Daten aus dem Backend", data))
```
Comment (fabianschmauder) [inline frontend/src/addEventPage/useEventMembers.js:19]: i would remove the comment

**Review by fabianschmauder — COMMENTED** [review]
LGTM small changes

**Pull request merged**